src/main/java/org/apache/commons/compress/archivers/zip/ParallelScatterZipCreator.java

```diff
@@ -17,13 +17,11 @@
  */
 package org.apache.commons.compress.archivers.zip;
 
-import org.apache.commons.compress.parallel.FileBasedScatterGatherBackingStore;
-import org.apache.commons.compress.parallel.InputStreamSupplier;
-import org.apache.commons.compress.parallel.ScatterGatherBackingStore;
-import org.apache.commons.compress.parallel.ScatterGatherBackingStoreSupplier;
+import org.apache.commons.compress.parallel.*;
```

Comment:
We do not use wildcard imports. Please specify the packages you want to import.

Reply:
> We do not use wildcard imports. Please specify the packages you want to import.

Good catch, I think Checkstyle can catch that.

Reply:
I have undone all .* imports in the whole zip package